ha/util/message_bus.py

```diff
@@ -144,6 +146,16 @@ def stop(self):
 
 class MessageBus:
     ADMIN_ID = "ha_admin"
+    _DELIM = ">"
+    HA_GLOBAL_INDEX = "ha_conf"
+
+    @staticmethod
+    def init():
+        """
+        Initialize utils MessageBus Library with kafka endpoints
+        """
+        message_server_endpoints = Conf.get(MessageBus.HA_GLOBAL_INDEX, f"kafka_config{MessageBus._DELIM}endpoints")
```

Comment:
This  should not have returned the correct endpoint . Can you please check ?
HA_GLOBAL_INDEX  is defined in const.py of HA and needs to be picked up from there

Reply:
As it is utils code keeping it separate from ha code, so in future if we need to move this code to utils it would be easy.
Already there are some const present in same file which are kept it here with same objective.

Reply:
Done. Used values from const.py